src/js/components/TabWelcome.js

```diff
@@ -15,12 +15,14 @@ import errors from "../errors"
 import ingestFiles from "../flows/ingestFiles"
 import initNewTab from "../flows/initNewTab"
 import refreshSpaceNames from "../flows/refreshSpaceNames"
+import menu from "../electron/menu"
 
 export default function TabWelcome() {
   let dispatch = useDispatch()
   let id = useSelector(Tab.clusterId)
   let spaces = useSelector(Spaces.getSpaces(id))
   let spacesPresent = spaces.length !== 0
+  const spaceContextMenu = menu.spaceContextMenu(id)
```

Comment:
I know this might be silly, but do you have any objection to using `let` for most variables like this? My reasons are simply that it's two less characters and makes code a little easier to read in my opinion. `const` will throw an error if it's re-assigned, which is a safeguard, but I've not had much benefit from that safety in my javascript time. Maybe I'm being superficial and if you feel strongly about `const` we can come to some consensus about where and when to use each.

Reply:
After chatting offline, we've decided to go with `const` for all things unless you need to use `let` for reassignment.